test/Microsoft.AspNetCore.Server.Kestrel.Performance/RequestParsing.cs

```diff
@@ -23,6 +23,11 @@ public class RequestParsing
 
         private const string plaintextRequest = "GET /plaintext HTTP/1.1\r\nHost: www.example.com\r\n\r\n";
 
+        private const string plaintextEmpower = "GET /plaintext HTTP/1.1\r\n" +
+            "Host: localhost\r\n" +
+            "Accept: text/plain,text/html;q=0.9,application/xhtml+xml;q=0.9,application/xml;q=0.8,*/*;q=0.7\r\n" +
+            "Connection: keep-alive\r\n\r\n";
```

Comment:
FYI, @davidfowl & @KrzysztofCwalina 

Looking at the TechEmpower code, it appears that their load generator doesn't send the Connection header (unless it somehow gets injected by the Python run-time later, which doesn't appear to be the case).